spotbugs-annotations/build.gradle

```diff
@@ -1,7 +1,5 @@
 apply from: "$rootDir/gradle/checkstyle.gradle"
 apply from: "$rootDir/gradle/javadoc.gradle"
-apply plugin: 'maven'
-apply plugin: 'signing'
```

Comment:
don't you need these plugins to be able to sign and publish to maven central the annotations artifacts?